docs/sdrkit/tools.mdx

```diff
@@ -363,7 +363,7 @@ Enables sales teams to manage and optimize multi-channel engagement.
 
               const prompt = await pull("hwchase17/openai-functions-agent");
 
-              const toolset = new LangchainToolSet({ apiKey: "npf6zezgodpzr3r19bdu9" });
+              const toolset = new LangchainToolSet({ apiKey: "<your-api-key>" });
```

Comment:
Good catch on removing the exposed API key. For better security practices, I recommend:
1. Immediately invalidate this API key if not done already
2. Consider implementing a secret scanning tool in the CI/CD pipeline to prevent future accidental exposures
3. Consider adding a note in the documentation about API key security best practices

Reply:
invalidated the API key